lib/connect_proto/src/substance.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+package primary.connect;
+option go_package = "primary.connect";
+
+import "google/protobuf/timestamp.proto";
+
+import "coded_value.proto";
+
+message Substance {
+  message Product {
+    CodedValue code = 1; // Code for the result component
+    string lot_number = 2;
+    google.protobuf.Timestamp expiration_date = 3;
+    CodedValue manufacturer = 4;
+  }
+
+  message Route {
+    CodedValue code = 1; // Code for the result component
+  }
+
+  message Site {
+    CodedValue code = 1; // Code for the result component
+  }
+
+  message Dose {
+    string quantity = 1;
+    string units = 2;
+  }
+
+  string notes = 1;
+  string refusal_reason = 2;
```

Comment:
Isn't this missing the use of the messages above?
```suggestion
  Product product = 1;
  Route route = 2;
  Site site = 3;
  Dose dose = 4;
  string notes = 5;
  string refusal_reason = 6;
```
As to `refusal_reason`, should this be some sort of coded value that we can normalize? Looks like Redox doesn't have it, so not really sure if there would be a fixed enum we could provide or some codset we could reference.

Reply:
Redox does have `Vaccinations[].RefusalReason`. I'll add those other messages.

Reply:
It looks like the code value makes sense from looking at the HL7 requirements  https://hl7-definition.caristix.com/v2/HL7v2.5.1/Fields/RXA.18 . I'll change that